subsys/net/lib/mqtt_sn/mqtt_sn.c

```diff
@@ -743,6 +820,12 @@ static int process_ping(struct mqtt_sn_client *client, int64_t *next_cycle)
 	return 0;
 }
 
+/**
+ * @brief Housekeeping task for the gateway search
+ *
+ * @param next_cycle will be set to the time when the next action is required
+ * @retval 0 on success
+ */
 static int process_search(struct mqtt_sn_client *client, int64_t *next_cycle)
```

Comment:
Just for the sake of completness - the `client` line is missing in doxygen. Although we don't render docs from `*.c` files anyway.

Reply:
I think we should fix the missing parameter even if we do not render these docs, people could still read the source file and it is helpful if the documentation is correct.

Reply:
Done - even though there is no deeper explanation still